packages/devtools_app/lib/src/flutter/connect_screen.dart

```diff
@@ -63,6 +63,7 @@ class _ConnectScreenBodyState extends State<ConnectScreenBody> {
               'Connect to a Running App',
               style: textTheme.bodyText1,
             ),
+            const Padding(padding: EdgeInsets.only(top: 6.0)),
```

Comment:
nit:  probably more idiomatic to wrap the Text with the padding if you want to visualize this as padding 
or
use
Container(height: 6)
if you just want a spacer.

Reply:
Thanks! Updated to use `const SizedBox(height: 6)`.